src/i18n/locales/fr.json

```diff
@@ -273,6 +273,10 @@
 		"SignIn": "Se connecter",
 		"SignOut": "Se déconnecter"
 	},
+	"Theme": {
```

Comment:
I'm not 100% sure these translations will actually be taken into account by Crowdin. What I've been doing is only change the en.json. We can leave it like this and test ^^

Reply:
Had to resolve a conflict in #929
![image](https://github.com/user-attachments/assets/d26dfb41-ed40-40d8-8469-fa05de0f780d)

==> Crowdin only looks at en.json

Reply:
Alright, good to know, thanks !